IPython/core/display.py

```diff
@@ -607,9 +620,19 @@ def data(self, svg):
             pass
         svg = cast_unicode(svg)
         self._data = svg
+    
+    def _data_and_metadata(self):
+        """shortcut for returning metadata with shape information, if defined"""
+        md = {}
+        if self.metadata:
+            md.update(self.metadata)
+        if md:
+            return self.data, md
+        else:
+            return self.data
 
     def _repr_svg_(self):
-        return self.data
+        return self._data_and_metadata()
```

Comment:
Does returning a 2-tuple from our repr methods actually work? I've never seen this before.